spec/acceptance/nodesets/archlinux-2-x64.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Would it be an idea to add these nodesets as unmanaged? Longer term I'd like to remove them and if this module isn't using them, let's not pollute its history.

Reply:
Sure.  Can you remind me of the correct syntax for `.sync.yml`?

Reply:
https://github.com/voxpupuli/puppet-windows_env/blob/542466bc58552befc8b209f0b317910c515cc9b3/.sync.yml#L4-L13